doc/classes/Viewport.xml

```diff
@@ -275,6 +275,9 @@
 		<member name="world_3d" type="World3D" setter="set_world_3d" getter="get_world_3d">
 			The custom [World3D] which can be used as 3D environment source.
 		</member>
+		<member name="use_xr" type="bool" setter="set_use_xr" getter="is_using_xr" default="false">
```

Comment:
It should be further up (alphabetical ordering).

Reply:
Geeeez, aren't we nitpicking today :P :P

Reply:
Done :)